src/templates/interface/video/floating.js

```diff
@@ -0,0 +1,51 @@
+import { slotService, utils } from '@wikia/ad-engine';
+import CloseButton from '../close-button';
+
+const FLOATING_CLASS_NAME = 'outstream-floating';
+
+function canFloat(params) {
```

Comment:
I think this function is redundant. Line 11 can be just `if (!params.isFloatingEnabled) {`

Reply:
Right, done